lib/tmuxinator/config.rb

```diff
@@ -62,7 +62,19 @@ def configs
         end
       end
 
-      def validate(name, options={})
+      def parse_settings(args)
+        settings = args.select { |x| x.match(/.*=.*/) }
+        args.reject! { |x| x.match(/.*=.*/) }
+
+        settings.map! do |setting|
+          parts = setting.split('=')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.